js/localStorage.js

```diff
@@ -0,0 +1,54 @@
+const formDataContent = {
+  available: null,
+  formName: document.getElementById('name'),
+  formEmail: document.getElementById('email'),
+  formMessage: document.getElementById('message'),
+  userData: {
+    name: null,
+    email: null,
+    message: null,
+  },
+  storedJson: null,
+  storedObject: null,
+  toStore: null,
+};
+
+if (window.localStorage) {
+  formDataContent.available = true;
+}
+
+const retrieveStorage = {
+  dataRetrieve() {
+    formDataContent.storedJson = localStorage.getItem('jsonObj');
+    formDataContent.storedObject = JSON.parse(formDataContent.storedJson);
+  },
+
+  attributes() {
+    formDataContent.formName.setAttribute('value', formDataContent.storedObject.formName);
+    formDataContent.formEmail.setAttribute('value', formDataContent.storedObject.formEmail);
```

Comment:
Kindly fix the error below. The form isn't working properly because of the error.

![image](https://user-images.githubusercontent.com/20554583/190839110-51307ebe-ee12-430c-9264-dd690125322a.png)

Please endeavor to make the data saving work properly as expected.